amr-wind/incflo.cpp

```diff
@@ -65,6 +65,9 @@ void incflo::init_mesh()
     } else {
         // Read starting configuration from chk file.
         ReadCheckpointFile();
+        for (int lev = finestLevel(); lev <= maxLevel(); ++lev) {
+            regrid(lev, m_time.current_time());
```

Comment:
Do we want to add some output here, e.g. "Regriding on restart" or something to that effect so that the user isn't confused by the checkpoint file and resulting possible mismatch in grid summary?

Reply:
Yeah, we can do that. I just figured that they had to manually change `max_level` in the input file. ;)

Reply:
Yes but maybe they want confirmation that it did actually regrid? I could go either way on this one.

Reply:
With the current version, the code does output the refinement levels right after reading the mesh.
Here is the output of the example I ran. 

`Restarting from checkpoint ../amrwind_abl/chk40000
Grid summary:
  Level 0   1024 grids  33554432 cells  100 % of domain
            smallest grid: 32 x 32 x 32  biggest grid: 32 x 32 x 32
  Level 1   800 grids  819200 cells  0.3051757812 % of domain
            smallest grid: 16 x 8 x 8  biggest grid: 16 x 8 x 8
  Level 2   648 grids  4810752 cells  0.2240180969 % of domain
            smallest grid: 24 x 16 x 16  biggest grid: 32 x 16 x 16
`

Reply:
@marchdf I'll merge this capability as it removes one of the impediments mentioned in #425. Based on user feedback, we can decide what additional informational messages we should add. Hope that is ok.